app/src/main/kotlin/amhk/chronos/database/ChronosDatabase.kt

```diff
@@ -21,11 +21,9 @@ internal abstract class ChronosDatabase : RoomDatabase() {
 
         @Synchronized
         fun get(context: Context): ChronosDatabase {
-            if (instance == null) {
-                instance = Room.databaseBuilder(context.applicationContext,
-                        ChronosDatabase::class.java, DATABASE_NAME)
-                        .build()
-            }
+            instance = instance ?: Room.databaseBuilder(context.applicationContext,
```

Comment:
Let's switch to the pattern in the google samples: https://github.com/googlesamples/android-architecture-components/blob/master/BasicRxJavaSampleKotlin/app/src/main/java/com/example/android/observability/persistence/UsersDatabase.kt

Reply:
+1
Want me to update the PR?